### PR TITLE
Redesigned the apps page to matrix style with on-hover display.

### PR DESCRIPTION
**Pull request by daminisatya**

### CODE REVIEW

File: html/apps/applist/style.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+p { margin: 0.5em 0; }
+.reveal p { margin: 0; }
+body { font-family: Lato, sans-serif; }
+.span2 p { text-align: center; }
+
+/* ===============================================
+// Reveal module
+// =============================================== */
+
+.reveal .hidden { display: block !important; visibility: visible !important; }
```
Comment (sudheesh001) [inline html/apps/applist/style.css]: Codacy Issue found: [Use of !important](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2611673807/issues/source?bid=3288751&fileBranchId=3310993#l14)

**Comment by daminisatya** [comment]
selection_003

**Comment by daminisatya** [comment]
loklak org - app list - google chrome_007

The on-hover details display

File: html/apps/applist/style.css (hunk 1 above)
Comment (daminisatya) [inline html/apps/applist/style.css]: @sudheesh001 can you be specific about the issue?

**Comment by daminisatya** [comment]
#356

**Comment by Orbiter** [comment]
great! good work!

**Comment by sudheesh001** [comment]
@daminisatya This looks great and the author links seem to be working but it'd be nicer to have if clicking on the tile would take it to the app too. Could you do that in the next pull request ? 😄

**Pull request merged**